src/network/networkd-lldp-rx.c

```diff
@@ -104,70 +102,3 @@ int link_lldp_rx_configure(Link *link) {
 
         return 0;
 }
-
-int link_lldp_save(Link *link) {
```

Comment:
hmm, but if this is gone, and networkd exits and returns, don't we lose valuable data?

Reply:
networkd removes the state files for saving LLDP neighbors on stop.